_rules/html-page-title-2779a5.md

```diff
@@ -3,7 +3,7 @@ id: 2779a5
 name: HTML page has title
 rule_type: atomic
 description: |
-  This rule checks that an HTML page has a title.
+  This rule checks that a non-embedded HTML page has a title. HTML pages embedded into other documents, such as through `iframe` or `object` elements are not applicable because they are not web pages according to the definition in WCAG.
```

Comment:
Yeah, I'd probably use "top-level browsing context" (or something in that line) instead of "non-embedded", given that it is the technical term of what we try to do. And similarly "nested browsing context" instead of "embedded".

OTOH, so far we've avoided links inside the description of rules.
This was fairly easy when descriptions were short; but might become increasingly harder with longer description (absorbing notes).

I do not know why we were avoiding links in descriptions and whether that still need to be the case or not…

Reply:
I think if we want to do that we should make the description  a heading in the doc, instead of making it part of the frontmatter. That's a separate PR. I don't think that should hold up this PR.

Reply:
Looking back at that specific case: how would you feel about moving this second sentence to the background section?
(https://github.com/act-rules/act-rules.github.io/issues/1305#issuecomment-644622078)

I feel that it is "information about the background for the development of the rule" (to quote the ACT format) and thus fits there.
Putting it in the description feels like it is very important information that we want to show immediately to readers. But in that case, it feels more like some "corner case FAQ" for people having read the rule and wondering "why only non-embedded?"